src/components/rankings/Rankings.js

```diff
@@ -3,7 +3,7 @@ import {Header} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
 import * as riderStageConnectionsActions from "../../actions/riderStageConnectionsActions";
 import store from "../../store";
-import {Route, Switch, Link, Redirect} from 'react-router-dom';
+import {Route, Switch, Link, Redirect, withRouter} from 'react-router-dom';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'Redirect'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)